stats.php

```diff
@@ -109,6 +117,46 @@ function track_file_delete() {
 	] );
 }
 
+function track_xml_rpc_password_type( $xmlrpc_method ) {
+	// Skip tracking for non-XML-RPC requests.
+	if ( ! defined( 'XMLRPC_REQUEST' ) || ! XMLRPC_REQUEST ) {
+		return;
+	}
+
+	// Skip tracking for unauthenticated requests.
+	if ( ! is_user_logged_in() ) {
+		return;
+	}
+
+	// Skip tracking for Jetpack requests.
+	if ( vip_is_jetpack_request() ) {
+		return;
+	}
+
+	XML_RPC_Auth_Tracker::track( $xmlrpc_method );
+}
+
+function application_password_did_authenticate( $user ) {
```

Comment:
```suggestion
function maybe_set_xml_rpc_auth_tracker_type( $user ) {
```
nitpick, not a blocker, but I think it would make it more clear when reading it.